alerts.tf

```diff
@@ -0,0 +1,80 @@
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+#
+# This software is the property of WSO2 LLC. and its suppliers, if any.
+# Dissemination of any information or reproduction of any material contained
+# herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+# You may not alter or remove any copyright or other notice from copies of this content.
+#
+# --------------------------------------------------------------------------------------
+
+module "email_channel_primary" {
+  source              = "git::https://github.com/wso2/gcp-terraform-modules.git//modules/gcp/Notification-Channel"
```

Comment:
Shall we create a tag from this repo and refer to a particular tag. This is applicable for all the source references.